skt/kerneltree.py

```diff
@@ -251,11 +253,11 @@ def checkout(self):
 
         # The git_cmd() method expects a list of args, not a list of strings,
```

Comment:
Can you change the comment to use the new method name (`__git_cmd()` instead of `git_cmd()`) as well?

Reply:
If you want to change the name in comment, could I create another PR to do that because there are a lot comment need to be changed including the previous PR (`KernelBuilder`)?

Reply:
You can add a commit for fixing all of the comments then, that would work too